### PR TITLE
chore(.github): PR template

### PR DESCRIPTION
**Pull request by devanshj**
Makes the template not use lists with checkboxes as github thinks those are tasks as it adds description like "1 of 3 Tasks completed" in the linked issues, PRs, etc

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -5,7 +5,8 @@
...
 
 #### Scope
-
-- [ ] Patch: Bug Fix
-- [ ] Minor: New Feature
-- [ ] Major: Breaking Change
+<!-- Pick one:
+Patch: Bug Fix
+Minor: New Feature
+Major: Breaking Change
```
Comment (gergelyke) [inline .github/PULL_REQUEST_TEMPLATE.md:11]:
```suggestion
```

Comment (gergelyke) [reply .github/PULL_REQUEST_TEMPLATE.md:11]: we might as well remove this, as we do not except breaking changes from contributors

Comment (devanshj) [reply .github/PULL_REQUEST_TEMPLATE.md]: I have committed the change although I think contributors should still be allowed to send PRs that have a breaking change (like a bug fix could be a breaking change), you then take a call whether to merge it when you release a major or reject it altogether.

Comment (gergelyke) [reply .github/PULL_REQUEST_TEMPLATE.md]: hm, that's a good point 🤔 I'd be happy to merge either way, let me know what you perefer

Comment (devanshj) [reply .github/PULL_REQUEST_TEMPLATE.md]: I guess we should keep as it was, I'll revert in that case

**Pull request merged**